internal/engine/actions/remediate/pull_request/pull_request.go

```diff
@@ -91,7 +90,7 @@ type paramsPR struct {
 func NewPullRequestRemediate(
 	actionType interfaces.ActionType,
 	prCfg *pb.RuleType_Definition_Remediate_PullRequestRemediation,
-	pbuild *providers.ProviderBuilder,
+	ghCli provifv1.GitHub,
```

Comment:
How will this look like once we generalize it for another provider, say GitLab?

Reply:
We will need a more generic trait at that point for pull requests.